hbase-hbtop/src/main/java/org/apache/hadoop/hbase/hbtop/screen/top/TopScreenModel.java

```diff
@@ -155,11 +158,13 @@ public boolean addFilter(String filterString, boolean ignoreCase) {
     }
 
     filters.add(filter);
+    decomposePushDownFilter();
```

Comment:
yeah, though need to reset push down filters but thought there is no harm to decompose in case user provides filter with drill-down fields.

Reply:
Let me check if there is no harm or not locally. I'm suspecting that when a user add a filter after drill-down in user/client mode, pushDownFilters will be cleared in decomposePushDownFilter(), which is a problem. Will run this feature locally and check that.

Reply:
It looks like it's causing a strange behavior as `pushDownFilters` will be cleared when adding a filter. I think we should remove to call decomposePushDownFilter() here.

Also, can you please show `pushDownFilters` in addition to normal filters in `FilterDisplayModeScreenView.showFilters()` so that users can see if pushDownFilters is set or not?